brave/brave-core
# Fixes hover card bubble chrome:// urls.

Fixes brave/brave-browser#6385

Replaces chrome:// scheme with brave://.

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Start Brave.
2. Navigate to brave://settings.
3. Hover mouse over the tab.
4. Observe popup showing the title Settings and subtitle brave://settings

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
